pkg/controller/controller.go

```diff
@@ -53,9 +54,10 @@ type Controller struct {
 	workflowStore       workflowstore.FlyteWorkflow
 	// recorder is an event recorder for recording Event resources to the
 	// Kubernetes API.
-	recorder      record.EventRecorder
-	metrics       *metrics
-	leaderElector *leaderelection.LeaderElector
+	recorder         record.EventRecorder
+	metrics          *metrics
+	leaderElector    *leaderelection.LeaderElector
+	namespaceFilterR *regexp.Regexp
```

Comment:
```suggestion
	namespaceFilterRegex *regexp.Regexp
```

Reply:
Will fix in a commit for code alignment sake.